### PR TITLE
Release notes + process bean

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major11/11.0.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -95,6 +95,20 @@ The following breaking changes were introduced:
...
 
+* **Deleted 'Authority' entities**
+  Valtimo has switched to a new policy based access control system. As a consequence, the old authorities have been
+  removed. The following entities can no longer be used:
+  - Database table: 'jhi_authority'
+  - Spring events: AuthorityCreatedEvent, AuthorityDeletedEvent, AuthorityEvent, AuthorityNameChangedEvent
+  - Spring beans: AuthorityResource, AuthorityService, AuthorityRepository
+  - Domain: Authority, AuthorityRequest, Money
+
+* **Deleted 'Context' entities**
```
Comment (marijnritense) [inline release-notes/major11/11.0.0/valtimo-backend-libraries.md:106]: Do we need something about this in the migration notes as well?

Comment (Klaas-Ritense) [reply release-notes/major11/11.0.0/valtimo-backend-libraries.md:106]: Can we have migration notes for this? Because there is no alternative

Comment (marijnritense) [reply release-notes/major11/11.0.0/valtimo-backend-libraries.md:106]: Something like "If you are using these entities/services, please don't use them anymore" I think still counts as a migration note.

File: reference/process-beans.md [hunk 2]
```diff
@@ -3,6 +3,28 @@
...
 
+## ConnectorService
+
+This process bean makes it possible to interact with connectors inside the BPMN process.
+
+```kotlin
+fun getConnectorTypes()
+fun getConnectorInstances(pageable: Pageable = Pageable.unpaged())
+fun getConnectorInstancesByType(typeId: UUID, pageable: Pageable = Pageable.unpaged())
+fun getConnectorInstancesByTypeName(typeName: String, pageable: Pageable = Pageable.unpaged())
```
Comment (marijnritense) [inline reference/process-beans.md:14]: I guess this is a little bit late, but shouldn't this either:
1. Consist of a single table  with the method in 1 column and the description in the second colum
2. Just be the list from line 20 to 26.

Right now you're listing the same thing twice for the most part.

File: release-notes/major11/11.0.0/migration.md [hunk 3]
```diff
@@ -47,6 +47,69 @@ This page describes how to update Valtimo from the previous version to the curre
         - AWS Parameter store
         - Kubernetes `.yaml` file
 
+* **Whitelist all services used in Camunda processes**
+
+  Scope: back-end
+
+    1. **List all services**
```
Comment (ivo-ritense) [inline release-notes/major11/11.0.0/migration.md:54]:
```suggestion
    1. **Find all services used in processes and decision tables**
```

File: reference/process-beans.md (hunk 2 above)
Comment (Klaas-Ritense) [reply reference/process-beans.md:29]: I changed it. But not to a table because that made the functions look bad

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**